packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -40,4 +41,16 @@ contract CoreElectionModule is ElectionStorage, OwnableMixin {
     function getMemberTokenAddress() public view returns (address) {
         return _electionStore().memberTokenAddress;
     }
+
+    function getNominees() external view returns (address[] memory) {
+        return AddressSet.values(_electionStore().nominees);
+    }
+
+    function selfNominate() external {
+        AddressSet.add(_electionStore().nominees, msg.sender);
+    }
+
+    function selfUnnominate() external {
+        AddressSet.remove(_electionStore().nominees, msg.sender);
```

Comment:
Do we want to revert if msg.sender is not in the set?

Atm it fails silently.

Reply:
Yup, done!